### PR TITLE
feat: expand image

### PR DESCRIPTION
**Pull request by cabreraalex**
# Description
Make images have an expand button and modal

https://github.com/zeno-ml/zeno-instance-views/assets/4563691/4c78421d-54fa-4cdf-a816-ef4659d32bce

### CODE REVIEW

File: src/lib/views/elements/Image.svelte [hunk 1]
```diff
@@ -1,5 +1,41 @@
...
+		<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24" class="w-6 fill-grey-darker">
+			<path d={mdiArrowExpand} />
+		</svg>
+	</button>
+</div>
+
+{#if showPopup}
+	<div
+		id="fullscreenModal"
+		class="bg-gray-500 fixed inset-0 z-50 flex items-center justify-center bg-opacity-75 p-4"
```
Comment (Sparkier) [inline src/lib/views/elements/Image.svelte:27]: Just a general note, we might have to check the z values at some point.

**Review by Sparkier — APPROVED** [review]
Haha what is that video? :D

File: src/lib/views/elements/Image.svelte [hunk 2]
```diff
@@ -1,5 +1,41 @@
...
+		<div class="relative w-full max-w-2xl rounded-lg bg-white p-6 shadow-xl">
+			<button
+				on:click={() => (showPopup = false)}
+				class="absolute right-0 top-0 mr-4 mt-4 rounded-md p-1 hover:bg-primary-ligther"
+			>
+				<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24" class="w-6 fill-grey-darker">
+					<path d={mdiClose} />
+				</svg>
+			</button>
+			<img src={data} alt="Image thumbnail for instance {data}" />
```
Comment (Sparkier) [inline src/lib/views/elements/Image.svelte:38]: What happens if the image is bigger than the canvas? Kinda edge case but might happen for big images.

**Review by Sparkier — APPROVED** [review]

File: src/lib/views/InstanceView.svelte [hunk 3]
```diff
@@ -72,9 +72,7 @@
 		{/if}
 	</table>
 {:else}
-	<div
-		class="min-w-[400px] max-w-[450px] overflow-x-scroll break-words rounded border border-grey-lighter p-4"
-	>
+	<div class="overflow-x-auto break-words rounded border border-grey-lighter p-4">
```
Comment (Sparkier) [inline src/lib/views/InstanceView.svelte:75]: It's easier to review these if they're based on main (or we have a review order). I looked at the width one last and wondered why it's in all of them. Generally if possible I think it's easiest if all PRs are based on main.

**Comment by cabreraalex** [comment]
> Haha what is that video? :D

lol didn't even check it, screen caps always lag to start for me ahaha

File: src/lib/views/InstanceView.svelte (hunk 3 above)
Comment (cabreraalex) [reply src/lib/views/InstanceView.svelte:75]: Aha true, but I was good and made these three separate PRs this time ;)

**Pull request merged**